execution_engine/src/storage/transaction_source/lmdb.rs

```diff
@@ -71,10 +73,12 @@ impl LmdbEnvironment {
         max_readers: u32,
     ) -> Result<Self, error::Error> {
         let env = Environment::new()
+            // Set the flag to manage our own directory like in the storage component.
+            .set_flags(EnvironmentFlags::NO_SUB_DIR)
             .set_max_dbs(MAX_DBS)
             .set_map_size(map_size)
             .set_max_readers(max_readers)
-            .open(path.as_ref())?;
+            .open(&path.as_ref().join("data.lmdb"))?;
```

Comment:
Can you make "data.lmdb" a `const &str` please?  I checked in components/storage.rs and see why you did it this way, but could you also change that too please so that "storage.lmdb" is a `const &str` in that file too?

Reply:
Will do!

Reply:
Added a `const &str`